qutebrowser/mainwindow/prompt.py

```diff
@@ -821,6 +842,7 @@ def _allowed_commands(self):
             ('prompt-open-download', "Open download"),
             ('prompt-open-download --pdfjs', "Open download via PDF.js"),
             ('prompt-yank', "Yank URL"),
+            ('prompt-external-picker', "External location picker"),
```

Comment:
```suggestion
            ('prompt-external-picker', "Launch external location picker"),
```

for consistency with the other descriptions.

Reply:
Done in 65c3007fc07f708e716c9cb14fefd10bf5f240d7